cmd/copyEnumeratorInit.go

```diff
@@ -102,6 +102,10 @@ func (cca *cookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 		cca.stripTopDir = true
 	}
 
+	if cca.fromTo.To() != common.ELocation.Blob() && cca.blobTags != nil {
+		return nil, fmt.Errorf("blob tags can only be set when transferring to blob storage")
+	}
```

Comment:
this check fits better in the cook function

Reply:
Ohh I was wondering about that too. Sure, I'll move it.